packages/server/utils/axios.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import { getAlfrescoUrlBase } from './alfresco';
+
+export const alfrescoAxios = axios.create({
+  baseURL: getAlfrescoUrlBase(),
+});
+
+alfrescoAxios.interceptors.response.use(
+  (response) => response,
+  (error) => {
+    if (axios.isAxiosError(error)) {
+      const simplifiedError = {
+        status: error.response?.status,
+        message: error.message,
+        stack: error.stack,
+      };
+      if (error.response?.status === 404) {
```

Comment:
I'm wondering if this 404 error logic applies elsewhere and not just in list-files. Or rather, if this should or should not be applied in general and not just there.

Reply:
Only applies to list-files, so it's not generic and could be removed here